src/cv/yolo_video_detect.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+import imutils
+import time
+import cv2
+import os
+
+def get_configs(args):
+    # Loads labels
+    labelsPath = os.path.sep.join([args["yolo"], "coco.names"])
+    labels = open(labelsPath).read().strip().split("\n")
+
+    # Create list of colors
+    np.random.seed(42)
```

Comment:
Replace this with a time here

Reply:
ok

Reply:
Later on we won't need to do this color generation at all